modules/nf-core/amrfinderplus/run/main.nf

```diff
@@ -43,7 +43,7 @@ process AMRFINDERPLUS_RUN {
         mkdir amrfinderdb
         tar xzvf $db -C amrfinderdb
     else
-        mv $db amrfinderdb
+        cp $db amrfinderdb
```

Comment:
What is the reason for moving from `mv` to `cp`?

Reply:
Maybe not important. Just when testing locally, my local folder was moved away from its location (where I actually wanted to keep it), so I thought `cp` is safer. But I don't have a strong opinion here.

Reply:
No strong opinion from me either. I trust your opinion since you have actually used the tool!